recipes/ade/all/test_package/test_ade.cpp

```diff
@@ -0,0 +1,4 @@
+#include "ade/graph.hpp"
+
+int main() {
```

Comment:
Please call some functions/symbols of ade.
A test package should test whether linking works.